db/migrate/20210223152748_add_operator_to_relative_date_filters.rb

```diff
@@ -0,0 +1,43 @@
+class AddOperatorToRelativeDateFilters < ActiveRecord::Migration[6.1]
+  def up
+    UserFilter.where('contents LIKE ?', '%relativeOption%').find_each do |user_filter|
+      filters = JSON.parse(user_filter.contents)
+      filters.each { |filter| add_operator_and_has_timestamp_to_filter(filter) }
+      filters.reject! { |filter| filter.dig('filterOption', 'hasTimestamp') && filter['value'].class == Hash && filter['value']['when'] == 'next' }
+      user_filter.update!(contents: filters.to_json)
+    end
+
+    UserExportPreset.where('config LIKE ?', '%relativeOption%').find_each do |user_export_preset|
+      config = JSON.parse(user_export_preset.config)
+      filters = config.dig('data', 'patients', 'query', 'filter')
+      filters.each { |filter| add_operator_and_has_timestamp_to_filter(filter) } if filters.present?
```

Comment:
How come we don't do a reject as well in this case for the UserExportPresets?

Reply:
ah, forgot to copy that logic over to UserExportPresets, will do that now

Reply:
https://github.com/SaraAlert/SaraAlert/pull/703/commits/ef87347e6c66a062b70c60b9b28ea86949d10237